README.md

```diff
@@ -91,13 +91,13 @@ specific RuboCop release.**
 
 **RuboCop**'s installation is pretty standard:
 
-```
+```sh
```

Comment:
True that. But on the other hand they make it crystal clear that something is a shell command.